cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/api/InstancePricingApi.java

```diff
@@ -31,6 +31,7 @@
             @Query( "cloudProvider" ) int cloudProvider, @Query( "deprecated" ) Boolean deprecated
     );
 
+    @Deprecated
     @GET( "instancePricing" )
     Call<List<InstancePricingDTO>> getCompatible(
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)